eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -5,7 +5,7 @@ parameters:
 stages:
   - stage: Build
     jobs:
-    - template: ../jobs/archetype-sdk-client.yml
+    - template: ../jobs/archetype-sdk-client.yml  
```

Comment:
nit: the only change to this file is an unnecessarily added space at the end of the line, please don't submit this if that's the only change